index/src/main/scala/org/dbpedia/spotlight/db/CreateSpotlightModel.scala

```diff
@@ -245,9 +253,9 @@ object CreateSpotlightModel {
     } else {
       new LanguageIndependentTokenizer(Set[String](), stemmer, locale, tokenStore)
     }
-    val fsaDict = FSASpotter.buildDictionary(sfStore, tokenizer)
+    //val fsaDict = FSASpotter.buildDictionary(sfStore, tokenizer)
```

Comment:
Good catch, I've put the line back in. It definitely should not break things, but actually this whole class is hard for me to test propery because there are so many dependencies. Does anyone have a setup in which they can easily build a new Spotlight model from scratch? I think it might be important to at least test this all the way through at some point.